kotlin/viewregistry-android/src/main/java/com/squareup/viewregistry/AlertContainer.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.viewregistry
+
+import android.content.Context
+import android.support.annotation.StyleRes
+import android.support.v7.app.AlertDialog
+import android.util.AttributeSet
+import android.view.ViewGroup
+import android.view.ViewGroup.LayoutParams.MATCH_PARENT
+import com.squareup.viewregistry.AlertScreen.Button.NEGATIVE
+import com.squareup.viewregistry.AlertScreen.Button.NEUTRAL
+import com.squareup.viewregistry.AlertScreen.Button.POSITIVE
+import com.squareup.viewregistry.AlertScreen.Event.ButtonClicked
+import com.squareup.viewregistry.AlertScreen.Event.Canceled
+import io.reactivex.Observable
+
+/**
+ * Class returned by [ModalContainer.forAlertContainerScreen], qv for details.
```

Comment:
qv?

Reply:
quod vide

https://english.stackexchange.com/questions/25252/how-does-one-correctly-use-q-v

It was _all over_ the Dungeons and Dragons books. 😁